LazyProxy/LazyProxyBuilder.cs

```diff
@@ -283,4 +283,4 @@ private static MethodInfo GetGetServiceValueMethod(FieldInfo serviceField)
             }
         }
     }
-}
\ No newline at end of file
+}
```

Comment:
It's not needed here.

Reply:
(All another files of this repo doesn't have EOF at the end of file)

Reply:
Not very good style. CLI git warns about it as well.
Removed the EOL for connsistency.